pinot-common/src/main/java/com/linkedin/pinot/common/metrics/ServerGauge.java

```diff
@@ -24,7 +24,8 @@
 */
 public enum ServerGauge implements AbstractMetrics.Gauge {
   DOCUMENT_COUNT("documents", false),
-  SEGMENT_COUNT("segments", false);
+  SEGMENT_COUNT("segments", false),
+  LAST_REALTIME_SEGMENT_FLUSH_DURATION_MILLIS("milliseconds", false);
```

Comment:
We should have multiple phases for this and document them. I don't think flush should include the completion protocol time, but as you say, segment creation duration is a better name for this. We'd also need a new metric for total time spent in the completion protocol.